### DISCUSSION TITLE
Does UUI components supports new electric skin?

### DISCUSSION THREAD

**Discussion by Hoviadin** [epam/UUI#1704] [Q&A] [answered]
Does UUI components supports electric skin? if yes, how I can use this skin?

**Comment by AlekseyManetov** [comment, marked as answer]
It's in development stage now, the stable release planned at the start of November. But if you need to start development right now, we can create a RC version to not to be a blocker for you.

**Reply by Hoviadin** [reply]
Thank you, we decided that we can wait for a stable release)
